Assets/Scripts/Photos/PhotoCamera.cs

```diff
@@ -113,56 +117,35 @@ public void Capture()
         cameraBrain.OutputCamera.targetTexture = null;
         foreach (Capturable capturable in capturables)
         {
-            if(capturable.IsVisibleOnCameraPlanes(orderedPlanes))
+            if (capturable.IsVisibleOnCameraPlanes(orderedPlanes))
             {
-                photo.AddIdentifiableObject(capturable);
+                photoDTO.AddIdentifiableObject(capturable);
             }
         }
 
-        // Adding photo collection and saving. SaveData() saves the whole collection so we should probably move this out somewhere else.
-        PhotoCollectionDTO.AddPhoto(photo);
-        PhotoCollectionDTO.SaveData();
+        imageData = photoDTO;
+        DisplayPhotoPreview();
 
-        StartCoroutine(this.DisplayPhoto());
     }
 
-    /// <summary>
-    /// Little short preview of the photo taken
-    /// </summary>
-    public IEnumerator DisplayPhoto()
+    void DisplayPhotoPreview()
     {
-        yield return new WaitForSeconds(1);
+        Cursor.lockState = CursorLockMode.None;
+        Cursor.visible = true;
```

Comment:
recommend to disable player movement's input gathering here (can set timescale to 0f, or introduce a new way in the player movement script) and re-enable when menu closed

Reply:
I introduced a static `TimeManager` class with some methods to check on pause state. Lemme know what you think